sonar-text-plugin/src/main/java/org/sonar/plugins/common/TextAndSecretsSensor.java

```diff
@@ -120,6 +101,10 @@ private static NotBinaryFilePredicate binaryFilePredicate(SensorContext sensorCo
     return new NotBinaryFilePredicate(sensorContext.config().getStringArray(TextAndSecretsSensor.EXCLUDED_FILE_SUFFIXES_KEY));
   }
 
+  private static TextFilePredicate textFilePredicate(SensorContext sensorContext) {
```

Comment:
TextFilePredicate doesn't explain what is happening here.

Reply:
I wanted to have it similar to its cousin.
`NotBinaryFilePredicate` -> predicate to get file that are considered as not-binary file (blacklist approach)
`TextFilePredicate` -> predicate to get file that are considered as text file (whitelist approach)
I will add comments to both methods to explain the purpose of each.